csrc/ops/composite.cpp

```diff
@@ -544,11 +544,24 @@ SdpfaFwdResult sdpfa_fwd(
   TensorView* log_sumexp =
       IrBuilder::create<TensorView>(log_sumexp_td, DataType::Float);
 
+  TensorView* philox_seed = nullptr;
+  TensorView* philox_offset = nullptr;
```

Comment:
nitpick: this is a `#if`. you don't need to declare/define nullptr before the macro.